marlowe-dashboard-client/src/MainFrame/Types.purs

```diff
@@ -91,7 +94,9 @@ data Action
   | ToggleCard Card
   | CloseCard
   | ContactAction Contact.Action
+  | ContractAction Contract.Action
   | ClickedButton
+  | StartContract Contract
```

Comment:
Should this be `LoadContract`? To be used when viewing a running/completed contract. `StartContract` can be for actually pushing a contract onto the blockchain (and then loading up to view as well).